src/PandoraMonitoring.cc

```diff
@@ -506,6 +513,26 @@ TEveElement *PandoraMonitoring::VisualizeCaloHits(const CaloHitList *const pCalo
         caloHitListName.replace(caloHitListName.find(starter), starter.length(), "CaloHits/");
     std::replace_if(caloHitListName.begin(), caloHitListName.end(), std::bind2nd(std::equal_to<char>(),'\n'), '/');
 
+    bool isU(false), isV(false), isW(false);
+
+    if (pCaloHitList->size() > 0)
```

Comment:
I tend to use 'if (!pContainer->empty())' as its reportedly faster (doesn't need to find the actual size; just check that there's something there). Not mandatory.

Reply:
Done.